dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageCheckerArguments.java

```diff
@@ -200,7 +202,13 @@ private static Options configureOptions() {
 
     if (commandLine.hasOption("b") || commandLine.hasOption("a")) {
       List<Artifact> artifacts = getArtifacts();
-      cachedInputClasspath = classPathBuilder.resolve(artifacts).getClassPath();
+      ClassPathResult result = classPathBuilder.resolve(artifacts);
+      ImmutableList<UnresolvableArtifactProblem> artifactProblems = result.getArtifactProblems();
+      if (!artifactProblems.isEmpty()) {
+        throw new RepositoryException(
+            "Unresolved artifacts: " + Joiner.on(",").join(artifactProblems));
```

Comment:
should probably add a space after the comma in the joiner

Reply:
Added